### PR TITLE
Submit page - external links open in new page, +spelling update

### LINKED ISSUE #318: [Bug] Submit a resource page - External links to github open on same page instead of new tab, nitpick: spelling of compare is incorrect
## Description
On the submit a resource page - external links to github website open on same tab. Ideally should open in a new tage

## Reproduction

1. Go to home page
2. Under resources section - click on 'submit one here'
3. Click on any external link like - creativecommons/cc-resource-archive - it'll open in same tab
4. Also, nitpick: compare is written as comapre which can be clubbed with the same issue

## Expectation

External links on this page should open in new tab

## Screenshots

https://github.com/user-attachments/assets/6deeaca7-ae4f-4512-bdc5-9efbdc7bc157

## Resolution

- [x] I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by mahak-juriani**
## Fixes

- Fixes #318  by @mahak-juriani

## Description

On submission page - opens github links in new page, spell fix

## Screenshots

https://github.com/user-attachments/assets/324fb542-cef4-4647-95e7-4293804fd43a

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: docs/_pages/submission.html [hunk 1]
```diff
@@ -48,7 +48,7 @@ <h3>
         You can submit your resource to Creative Commons through a Pull Request
         to the
         <a href="https://github.com/creativecommons/cc-resource-archive/"
-          >creativecommons/cc-resource-archive</a
+          target="_blank">creativecommons/cc-resource-archive</a
```
Comment (Murdock9803) [inline docs/_pages/submission.html:51]: @mahak-juriani After all the changes you have done, you should format the file with `prettier code formatter`. 
More context: [Formatting with prettier](https://github.com/creativecommons/cc-resource-archive?tab=readme-ov-file#formatting-with-prettier)

Comment (mahak-juriani) [reply docs/_pages/submission.html]: @Murdock9803 Thank you for the suggestion. I have updated the file accordingly. Please review

**Comment by Murdock9803** [comment]
The pull request looks good 🙌
@possumbilities we can move forward with this 🙌

**Review by possumbilities — APPROVED** [review]
Wondeful! thankyou! Approved.

**Review by Murdock9803 — COMMENTED** [review]
Thank you for the contribution 🙌

**Pull request merged**